cmd/eval-dev-quality/cmd/evaluate.go

```diff
@@ -46,6 +46,8 @@ type Evaluate struct {
 	Repositories []string `long:"repository" description:"Evaluate with this repository. By default all repositories are used."`
 	// ResultPath holds the directory path where results should be written to.
 	ResultPath string `long:"result-path" description:"Directory path where results should be written to. The placeholder \"%datetime%\" can be used for the current date and time." default:"evaluation-%datetime%"`
+	// Retries holds the number of retries to perform when a model errors.
+	Retries uint `long:"retries" description:"Number of retries to perform when a model errors." default:"2"`
```

Comment:
nit: `Number of retries to perform when a model errors in the process of solving a task.`